src/helm-frontend/tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "useDefineForClassFields": true,
+    "lib": ["ES2020", "DOM", "DOM.Iterable"],
+    "module": "ESNext",
+    "skipLibCheck": true,
+
+    "moduleResolution": "bundler",
+    "allowImportingTsExtensions": true,
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": true,
+    "jsx": "react-jsx",
+
+    "strict": true,
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "noFallthroughCasesInSwitch": true,
+
+    "baseUrl": "src",
+    "paths": {
+      "@/*": ["*"]
+    },
+
+    "types": ["vitest/globals", "vite/client"]
+  },
+  "include": ["src"],
+  "references": [{ "path": "./tsconfig.node.json" }]
```

Comment:
I'm a bit confused how the `references` field here works, and also the `include` field in `tsconfig.node.json`... could you give a quick explanation or link for my benefit? Or is this auto-generated scaffolding?

Reply:
This does appear to be part of the template when creating the project:

https://github.com/vitejs/vite/blob/ab5c421571485df861119e1f2366a84cc12b212d/packages/create-vite/template-react-ts/tsconfig.json#L24

I am not sure why the node specific config needs to referenced.

Reply:
Ah, if it's auto-generated, then it should be okay. Maybe I will investigate further later.